web-dev/src/components/LoginPage/index.js

```diff
@@ -1,111 +1,88 @@
+      alert("Invalid emailOrNumber/password");
+    });
+    event.preventDefault();
+  }
+  return (
+    <UnauthenticatedLock>
+      <div className="login-background flex-container"
+           style={{backgroundImage: `url(${LoginBackground})`}}>
+        <div className="login-bring-it-forward">
+          <img className="login-logo-size" src={require('./netflicks-logo.png')} alt="Logo" />
+          <div className="login-main-frame">
+            <div>
+              <div className="login-main-subframe">
+                <div className="login-main">
+                  <h1 className="login-title">Sign In</h1>
+                  <form onSubmit={handleSignin}>
+                    {/*Email or Phone number*/}
+                    <div className="login-user-frame">
+                      <div className="position-relative">
+                        <div className="login-email-box">
+                          <label htmlFor="login-user-input"
+                                 className="login-user-label-title text-white-50">
+                            Email or phone number
+                          </label>
+                          <input type="text"
+                                 ref={emailRef}
+                                 className="login-user-input text-white"
+                                 id="login-user-input"
+                                 placeholder="Email or phone number"
+                                 required
+                          />
                         </div>
+                      </div>
                     </div>
+                    {/*Password*/}
+                    <div className="login-user-frame">
+                      <div className="position-relative">
+                        <div className="login-email-box">
+                          <label htmlFor="login-password"
+                                 className="login-user-label-title text-white-50">
+                            Password
+                          </label>
+                          <input type="password"
+                                 ref={passwordRef}
+                                 autoComplete="on"
+                                 placeholder="Password"
+                                 className="login-user-input text-white"
+                                 id="login-password"
+                                 required
+                          />
+                        </div>
+                      </div>
+                    </div>
+                    <button className="btn login-button btn-submit btn-small"
+                            type="submit">
+                      Sign In
+                    </button>
+                    <div className="mt-3 mb-3">
+                      <Link to="/registration" className="login-registration-link text-white-50">
+                        New to Netflicks?
+                        <span className="text-white"> Sign up now. </span>
+                      </Link>
```

Comment:
We no longer allow guest so I took the liberty of removing it

Reply:
The link allows guests to go to the homepage right? Guests should be able to navigate to the homepage per the project requirements since the guest needs to be able to search for movies from the search bar. But we won't let them look at movie details.